lib/smart_udap_harmonization_test_kit/metadata.rb

```diff
@@ -0,0 +1,42 @@
+require_relative 'version'
+
+module SMART_UDAP_HarmonizationTestKit
+  class Metadata < Inferno::TestKit
+    id :smart_udap_harmonization
+    title 'SMART-UDAP Harmonization Test Kit'
+    description <<~DESCRIPTION
+      The [Security for Scalable Registration, Authentication, and Authorization
+      IG](https://hl7.org/fhir/us/udap-security/index.html) states that
+
+      > This guide is also intended to be compatible and harmonious with client and
+      > server use of versions 1 or 2 of the HL7 SMART App Launch IG.
+
+      This test kit is an effort to demonstrate how a client could interact with a
+      server supporting both UDAP and SMART App Launch.
+      <!-- break -->
```

Comment:
The break should be before line 8, and before break should be:
> The SMART-UDAP Harmonization Test Kit is an experimental Test Kit for evaluating options for testing authorization systems that conform to both SMART App Launch Implementation Guide and the Security for Scalable Registration, Authentication and Authorization Implementation Guide requirements.

unless it was meant to be changed?

Reply:
before line 8 is the constant `DESCRIPTION`, do you mean line 10?